brainpy/_src/math/ndarray.py

```diff
@@ -1033,57 +1038,43 @@ def addr(self,
       vec2 = Array(vec2)
     if not isinstance(out, Array):
```

Comment:
Remove this checking, because it will not work. 

``out`` must be a brainpy Array. It will be check whether it is ``brainpy.math.Array`` when we call ``brainpy.math.outer``.